### DISCUSSION TITLE
Some questions about TextPredictor.fit()

### DISCUSSION THREAD

**Discussion by icarosun** [autogluon/autogluon#2103] [Q&A] [answered]
- What means epoch in Fit? 
  * How epoch is slice?
- Does it used KFolds in train? 
- How can I get the val accuracy at fit? 

*I would like too much, if you can suggestion a papper about TextPredictor.

# **Thanks!**

**Comment by sxjscience** [comment, marked as answer]
1. What means epoch in Fit? : For one epoch, we mean to run a whole pass of the training data.
2. Does it use KFolds in train: No, it does not. It will split the validation dataset automatically for you. If you'd like to use K-Fold, you can do it in an outside loop.
3. How can I get the val accuracy at fit? : You can call `predictor.evaluate()`.
4. Paper: https://arxiv.org/abs/2111.02705
